apps/storybook/src/components/ai-message-bubble/index.tsx

```
@@ -0,0 +1,318 @@
+          {bubbleContent}
+        </motion.div>
+      );
+    }
+
+    return bubbleContent;
+  }
+);
+
+AIMessageBubble.displayName = "AIMessageBubble";
+
+export { AIMessageBubble, bubbleVariants };
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)